readalongs/align.py

```diff
@@ -956,54 +968,61 @@ def get_ancestor_sent_el(word_el: etree.ElementTree) -> Union[None, etree.Elemen
     return word_el
 
 
-def get_words_and_sentences(results) -> Tuple[List[dict], List[List[dict]]]:
-    """Parse xml into word and sentence 'tier' data
+def get_textual_words_and_sentences(
+    words: List[dict], tokenized_xml: etree.ElementTree
+) -> Tuple[List[dict], List[List[dict]]]:
+    """Parse xml into word and sentence 'tier' data with full textual words
```

Comment:
What are "textual" words? I find this a bit confusing. I see. They're the actual orthographic words, not our internal ID representation. I would maybe change this to `get_orthographic_words_and_sentences`? I don't know, our internal format is still text, but it's not orthographic (for the language).

Reply:
Right, that would be clearer. I wanted to distinguish from ID, and I agree "orthographic" more precisely makes that distinction.

Reply:
both are fairly clumsy to my ears/eyes, would recommend something like `get_word_names_and_sentences`, and `word_name` instead of `textual_word`

Reply:
Hum, I'm not sure I like "name" better. That doesn't ring "the word written out" to me.
But I do really like the idea of changing `"text"` as the field name to `"id"` in the case of word IDs, and maybe keep `"text"` for orthographic words, or `"ortho"` or maybe something else.

Reply:
how about `get_word_texts_and_sentences`?  mainly I just think `orthographic` and `textual` are much too verbose, and `ortho` is the name of a pesticide, not an API call :)

Reply:
Yeah, I like `get_word_texts_and_sentences`, `word["text"]`, `word_text`

Reply:
I just realized another nice thing about using just `text` is that it makes it consistent with the lxml.etree object these comes originally come from, since the text is (mostly) in the `.text` field of the etree object.